editoast/src/core/pathfinding.rs

```diff
@@ -73,9 +76,9 @@ pub struct InvalidPathItem {
     pub path_item: PathItemLocation,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, ToSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
 #[serde(tag = "status", rename_all = "snake_case")]
-pub enum PathfindingResult {
+pub enum PathfindingCoreResult {
```

Comment:
We already in the `core` module ;)

```suggestion
pub enum PathfindingResult {
```

Reply:
PathfindingResult is already defined :(